src/views/BreadView/BreadTable.js

```diff
@@ -0,0 +1,178 @@
+import IconButton from '@material-ui/core/IconButton';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import AddIcon from '@material-ui/icons/Add';
+import ClearIcon from '@material-ui/icons/Clear';
+import LockIcon from '@material-ui/icons/Lock';
+
+import AddBreadDialog from './AddBreadDialog';
+
+import breadTableStyle from '../../assets/jss/breadTableStyle';
+import * as api from '../../api';
+import * as utils from '../../utils';
+
+const emptyRow = (classes, message) => {
+    return (
+        <TableRow>
+          <TableCell className={classes.dateCell}>{message}</TableCell>
+          <TableCell className={classes.descriptionCell}></TableCell>
+          <TableCell className={classes.buttonCell}>
+            <IconButton title="Lock" disabled>
+              <LockIcon fontSize="small" />
+            </IconButton>
+          </TableCell>
+        </TableRow>
+    )
+};
+
+
+class BreadTable extends React.Component {
+
+  state = {
+    dialogOpen: false,
+    submitting: false,
+    order: {},
+    items: {},
+    selectedItem: "buh"
+  }
+
+  handleAddClick = (order) => () => {
+    this.setState({dialogOpen: true, order: order});
+  }
+
+  handleClearOrder = (order) => () => {
+      this.setState(
+          {dialogOpen: false, order: order},
+          () => this.doClear()
+      );
+  }
+
+  doClear = () => {
+    api.del({
+        path: '/bread/' + this.state.order.id,
+    }).then(data => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage('Bestelling verwijderd', this.props.refresh)
+        );
+    }).catch(error => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage(error.message)
+        );
+    })
+  }
+
+  handleAddToOrder = value => {
+    this.setState(
+        {dialogOpen: false, submitting: true},
+        () => this.doAddItem(value)
+    );
+  }
+
+  doAddItem = value => {
+    api.patch({
+        path: '/bread/' + this.state.order.id,
+        data: { items: [value] },
+    }).then(data => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage('Brood toegevoegd', this.props.refresh)
+        );
+    }).catch(error => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage(error.message)
+        );
+    })
+  }
+
+  handleDialogChange = (dialogOpen) => () => {
+    this.setState({dialogOpen: dialogOpen});
+  }
+
+  render () {
+    const { classes, orders, loading, items } = this.props;
+
+    return (
+        <React.Fragment>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell className={classes.dateCell}>Week</TableCell>
```

Comment:
Week -> Besteldatum?

Reply:
Het is wel degelijk de week die ik bedoel, de week van "datum"

Reply:
Misschien is 'leverdatum' dan duidelijker? Anders kan je het ook interpreteren als 'ergens in de week van de dag die staat aangeduid zal het geleverd worden'

Reply:
Leverdatum kan ook wel, wat dacht je van "de week van"?